Tasks/DeployAzureResourceGroup/AzureUtilityLTE9.8.ps1

```diff
@@ -153,7 +153,7 @@ function Get-AzureRMVMsInResourceGroup
             $azureVMResources = Get-AzureVM -ResourceGroupName $resourceGroupName -ErrorAction Stop -Verbose
             Write-Verbose -Verbose "[Azure Call]Count of resource group:$resourceGroupName RM virtual machines type resource is $($azureVMResources.Count)"
         }
-        catch [Microsoft.WindowsAzure.Commands.Common.ComputeCloudException],[System.MissingMethodException], [System.Management.Automation.PSInvalidOperationException]
+        catch [Microsoft.WindowsAzure.Commands.Common.ComputeCloudException],[System.MissingMethodException], [System.Management.Automation.PSInvalidOperationException], [Hyak.Common.CloudException]
```

Comment:
you can remove [Microsoft.WindowsAzure.Commands.Common.ComputeCloudException] it. Also since Hyak is parent exception is there a scenario where any other child exception can fall here. so Should We check $_.exception.type after catching and check whether it is computecloudexception ?